packages/augur-sdk/src/Augur.ts

```diff
@@ -161,7 +161,7 @@ export class Augur<TProvider extends Provider = Provider> {
   }
 
   bindTo<R, P>(f: (db: any, augur: any, params: P) => Promise<R>): (params: P) => Promise<R> {
-    return Augur.connector.bindTo(f);
+    return Augur.connector && Augur.connector.bindTo(f);
```

Comment:
@bthaile Why did you need to do this?

Reply:
for some reason bindTo was called when `connector` was null. it was in a test.

Reply:
Create needs to be called first. I wonder what we were doing where that wasn't happening.